src/features/import/hooks/useDateConfig.ts

```diff
@@ -54,7 +54,12 @@ export default function useDateConfig(column: DateColumn, columnIndex: number) {
 
   const personNumberFormats: PersonNumberFormat[] = [];
 
-  if (organization) {
+  if (
+    organization &&
+    (organization.country === 'se' ||
+      organization.country === 'dk' ||
+      organization.country === 'no')
+  ) {
```

Comment:
There's nothing wrong (in general, ignoring the specific lowercase/uppercase bug for now) with an expression like this, but there are two things that I would do differently.

## The use of `===`
First of all, I never use `===`. The purpose of that operator is to make sure in javascript that two values are not just "semantically" equal but also technically equal. For example, `1 == '1'` evaluates to true, but `1 === '1'` evaluates to false. You probably know this and feel that the stricter comparison is better because it's stricter.

But the thing is, the reason the `===` operator was added ([in hindsight](https://stackoverflow.com/a/53111225/229824)) to javascript was because of javascripts very weak typing. In most other programming languages `1 == '1'` would already evaluate to false, so `===` would not be necessary. And because we use typescript where types are stricter, we should never end up in a situation where we accidentally compare two values of different types. If you do try to do `1 == '1'` in typescript, you will actually [get an error](https://www.typescriptlang.org/play/?#code/IwAgvGIOTFQ). 

So I prefer to always use `==` because to me it signals to the reader that we _know_ the two operands are the same type, whereas using `===` signals to the reader an insecurity about the types on the part of the coder, which we should never have to feel.

## Many ORs
This is another matter of personal taste for me, but I like to avoid having lots of `||` in an `if` statement. Sometimes they can be replaced by some other expression (like here) and in other cases the ORs are necessary but then they can at least (always) be broken out and named.

### Using `Array.prototype. includes`
The same logic could be expressed as:

```suggestion
  if (
    organization && 
    ['se', 'dk', 'no'].includes(organization.country)
```

In fact, the whole expression could be abbreviated even more by using the optional chaining operator (`?.`) instead of explicitly checking whether `organization` is truthy:

```
  if (['se', 'dk', 'no'].includes(organization?.country)
```

This is just a shorter way of expressing the same thing, and although it does remove the need for having so many line breaks, it isn't necessarily easier to read. In some situations I would argue that it has an edge, because it avoids having _nested_ booleans like you have here (`A && (B || C || D)` becomes just `A && E`) but the expression itself could still be argued is more difficult to interpret, which is why it sometimes makes even more sense to name the expressions.

### Naming expressions (variables)
Most would very intuitively identify when code could be broken out into a function to make it more readable. One of the benefits of doing so is that the function will get a name, and the call site will be easier to read because the logic in the function is expressed as the function's name.

But the same principle can be applied to simpler expressions (like this one) using just variables. If we give the second operand of the `&&` operator a name, the if statement will be easier to read:

```suggestion
  const isNordicCountry = (organization.country === 'se' ||
      organization.country === 'dk' ||
      organization.country === 'no');
  if (organization && isNordicCountry) {
```

I realize this exact example doesn't really make much sense because the reason the `organization` expression is there is to make sense that `organization.country` doesn't throw an error. Again, there's nothing objectively wrong with what you've already done and I'm not suggesting you change anything. I'm just making a general comment on code structure that I hope can be of value.

Reply:
I honestly think that now that we have had this bug behaviour, that the UX this was trying to create (being "kind" to the organiser by only showing the most likely option) is not, I think, a great idea, and does not solve a real world observed problem. So honestly I think the most reasonable thing to do is to just let everyone always see all the three options of person numbers. It's not hard to pick one of three, and if for some odd reason a Danish org would like to import people with Swedish personnummer, they can do so.

Reply:
Yes, the reason I lean to use `===` is that it is somewhere ingrained in me from being in Javascript that I don't trust `==`. It is more out of habit that I keep using it even in Typescript. Makes sense to be observant of this and trust the framework of Typescript to do what it is built to do 😸